README.md

```diff
@@ -1,13 +1,13 @@
-# Project Name
+# Noemi's Vinyl collection
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+I've been wanting to list my vinyls in some way for quite a while now and it just seemed to suit this flexbox-excercise.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+The flexbox-model worked out quite well i guess. For the phone screen it's still quite small and i'm not quite happy with the design yet. I'll work more on that. The website I published through netlify took a while to load all pictures, but I think I could work that out by scaling down the sizes of the images.
+I also wanted to include a linked overview (A-Z) so I can jump directly to artists in the end of the alphabet. I could solve this by linking the letters A-Z to the referring flex-box-child. It turned out like I imagined it, but not sure if this is the easiest and clearest way to go. I wish I could include a linked box to the top of the website, that is fixed to the site even when scrolling, but couldn't make that work.
+What I also wanted to try out was a hovering effect. So that when the curser hovers over the album cover, the artist and albumtitle turn up. But I couldn't figure that out. It destroyed the whole flexbox-style I've been working on for quite a while, so I deleted it again:) 
```

Comment:
Here you can check out `tooltip` which shouldn't mess with your other elements: https://www.w3schools.com/css/css_tooltip.asp